lib/soap.ts

```diff
@@ -151,9 +151,9 @@ function parameterValueList(
         }
       }
 
-      const valueType = getValueType(valueElement.attrs);
+      const valueType = valueElement&&valueElement.attrs?getValueType(valueElement.attrs):'xsd:string';
```

Comment:
my customized cpe does not include attrs field in valueElement so add some protection here

Reply:
This needs to be put behind a configuration option (disabled by default). Feel free to submit a separate PR for this.